### PR TITLE
#569 'Room null' is displayed for employees without rooms

### PR DESCRIPTION
**Pull request by YaroTs**

### CODE REVIEW

File: mobile/src/reducers/organization/employee.model.ts [hunk 1]
```diff
@@ -99,16 +99,26 @@ export class Employee {
...
 
+    //----------------------------------------------------------------------------
     public getName(): string {
         const names = this.name.split(' ');
 
         return names.length > 1 ? names[1] : this.name;
     }
 
+    //----------------------------------------------------------------------------
+    public getRoomTitle(): string {
```
Comment (DesmanLead) [inline mobile/src/reducers/organization/employee.model.ts]: I'm a bit concerned that there are 2 places with room number string generation. Maybe it's better to extract this somewhere?

Comment (YaroTs) [reply mobile/src/reducers/organization/employee.model.ts]: Sounds reasonable, but where?

Comment (DesmanLead) [reply mobile/src/reducers/organization/employee.model.ts]: That's a good question. Some kind of RoomFormatter?

Comment (antonm76) [reply mobile/src/reducers/organization/employee.model.ts]: Maybe pass fully constructed room title into CurrentPeopleRoomImpl as a nav param would be a solution? I see CurrentPeopleRoomImpl is the only place where room title is constructed besides this function

Comment (YaroTs) [reply mobile/src/reducers/organization/employee.model.ts]: You should still provide some default value if there is no required parameter in payload in this case

Comment (YaroTs) [reply mobile/src/reducers/organization/employee.model.ts]: @DesmanLead Take a look at latest changes, please

**Pull request merged**